lookout/style/format/analyzer.py

```diff
@@ -29,6 +29,20 @@
 from lookout.style.format.utils import generate_comment, merge_dicts
 from lookout.style.format.virtual_node import VirtualNode
 
+FixData = NamedTuple("FixData", (
+    ("error", str),                           # error message
+    ("language", str),                        # programming language of the code
+    ("line_number", int),                     # line number for the comment
+    ("head_file", File),                      # file from head revision
+    ("base_file", File),                      # file from base revision
+    ("suggested_code", str),                  # code line predicted by our model
+    ("vnodes", List[VirtualNode]),            # VirtualNode-s which correspond to this line
+    ("fixes", List[Tuple[int, int]]),         # changed vnode indices (in `vnodes`)
```

Comment:
@vmarkovtsev, Let's convert `Tuple[int, int]` to another `NamedTuple` ?

Reply:
I have just understood that we don't need vnode indices at all: we can scan through each vnode in `vnodes` and `getattr(vnode, "y_old") != vnode.y`

Reply:
So `fixes` can be renamed to `winner_rules`

Reply:
also, we should check line number, but yes, it is possible. just requires linear time.